components/layout/Container.js

```diff
@@ -6,15 +6,19 @@ export default function Container ({
   flexReverse = false,
   cssBreakingPoint = 'md',
   className = '',
+  tag,
 }) {
   const commonClassNames = `${flex ? `${cssBreakingPoint}:flex` : 'block'} ${flexReverse ? `${cssBreakingPoint}:flex-row-reverse` : ''} ${className} px-4 sm:px-6 lg:px-8`
   const wideClassNames = `max-w-screen-xl ${commonClassNames}`
   const regularClassNames = `max-w-4xl ${commonClassNames}`
   const normalClassNames = `${wide ? wideClassNames : regularClassNames} mx-auto w-full`
   const fluidClassNames = `${commonClassNames}`
+
+  const Tag = tag || 'div';
+  
   return (
-    <div className={fluid ? fluidClassNames : normalClassNames}>
+    <Tag className={fluid ? fluidClassNames : normalClassNames}>
```

Comment:
should this be left as `<Tag ....>`?

Reply:
if you pass `as` prop then you should use `Tag`